Gordon360/Services/PosterCleanupService.cs

```diff
@@ -0,0 +1,66 @@
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Gordon360.Services;
+using System.Linq;
+
+
+
+
+public class PosterCleanupService : BackgroundService
+{
+    private readonly ILogger<PosterCleanupService> _logger;
```

Comment:
```suggestion
    //private readonly ILogger<PosterCleanupService> _logger;
```
Comment out with rest of logging code.  (Or delete it all.  Usually we delete commented out code, but in this case it might be useful to have notes about where to log, if we decide it's needed.)